.gitignore

```diff
@@ -1,2 +1,11 @@
 node_modules
-/lib
\ No newline at end of file
+/lib
+
+# IDEs and editors
+/.idea
+.project
+.classpath
+.c9/
+*.launch
+.settings/
+*.sublime-workspace
```

Comment:
You can use a local, global ~/.gitignore for your own tool related stuff, a good practice is to put only build related ignores in a given project.

Reply:
Sure. I can remove those.
Just as a separate question:
> Wouldn't that make life easier for other developers who would join development contributions in the future? So that they would not need to discover, that "Ah, my IDE sees some workspace related files, that I don't want to accidentally add to commit"

Reply:
Well, I don't mind if there are some lines in the `.gitignore` since they is a comment to explain what they are related to.